From aadfb88a2f27ebf4c5590aed83433be7822655a6 Mon Sep 17 00:00:00 2001
From: kas User <kas@example.com>
Date: Fri, 20 Sep 2024 17:02:59 +0100
Subject: [PATCH] Same absl as yocto scarthgap

---
 third_party/absl/workspace.bzl | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/third_party/absl/workspace.bzl b/third_party/absl/workspace.bzl
index 06f75166ce4..bc2231601f9 100644
--- a/third_party/absl/workspace.bzl
+++ b/third_party/absl/workspace.bzl
@@ -7,8 +7,8 @@ def repo():
 
     # Attention: tools parse and update these lines.
     # LINT.IfChange
-    ABSL_COMMIT = "fb3621f4f897824c0dbe0615fa94543df6192f30"
-    ABSL_SHA256 = "0320586856674d16b0b7a4d4afb22151bdc798490bb7f295eddd8f6a62b46fea"
+    ABSL_COMMIT = "d7aaad83b488fd62bd51c81ecf16cd938532cc0a"
+    ABSL_SHA256 = "3a1f642ea21f45b220a01708d06b46042a3b750f31772b52371d6268f02d5bbc"
     # LINT.ThenChange(//tensorflow/lite/tools/cmake/modules/abseil-cpp.cmake)
 
     SYS_DIRS = [
